src/main/java/com/google/sps/data/Event.java

```diff
@@ -19,6 +21,7 @@ public Event(EventBuilder builder) {
     this.eventId = builder.eventId;
     this.eventName = builder.eventName;
     this.dateTime = builder.dateTime;
+    this.startDateTime  = builder.startDateTime;
```

Comment:
What's the distinction between dateTime and startDateTime? Please add an implementation comment to clarify.